starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,55 @@
+* {
+    font-family: Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+}
+body {
+    margin: 0px
+}
+.header-site  {
+    background-image: url('../images/apple-pie.jpg');
+    background-repeat: no-repeat;
```

Comment:
add background-size: cover; and background-position: center; and the image will be perfect!